tests/system/test_unreachable.py

```diff
@@ -0,0 +1,91 @@
+""" Test using root marathon.
+    This test suite is used to test unreachable strategy configurations on live clusters.
+"""
+
+import apps
+import common
+import json
+import os
+import pytest
+import retrying
+import shakedown
+import time
+import uuid
+
+from dcos.mesos import DCOSClient
+from dcos import marathon, errors, mesos
+from datetime import timedelta
+
+
+from shakedown import dcos_version_less_than, marthon_version_less_than
+from fixtures import wait_for_marathon_and_cleanup
+
+
+@pytest.fixture(scope="function")
+def marathon_service_name():
+    return "marathon"
+
+
+def setup_module(module):
+    common.cluster_info()
+
+
+def teardown_module(module):
+    common.clean_up_marathon()
+
+# This can not be run currently as `shakedown -q tests/system/test_unreachable.py`
+# They currently need a 20 min delay prior to running each one separately or you get
+# inappropriate results.
+# TO RUN:  shakedown -q tests/system/test_unreachable.py::test_unreachable_within_inactive_time[360-360]
+@pytest.mark.parametrize("inactive_sec, expunge_sec", [
+  (0, 0),
+  (60, 60),
+  (360, 360),
+  (60, 360),
+  (600, 600),
+  (1200, 1200)])
+def test_unreachable_within_inactive_time(inactive_sec, expunge_sec):
+
+    common.clean_up_marathon()
+    app_def = apps.unreachable()
+    app_id = app_def['id']
+
+    app_def = common.unreachableStrategy(app_def, inactive=inactive_sec, expunge=expunge_sec)
+    print(app_def)
+
+    client = marathon.create_client()
+    client.add_app(app_def)
+    shakedown.deployment_wait()
+    tasks = client.get_tasks(app_id)
+
+    # confirm task and get task_id and host details.
+    assert len(tasks) == 1, "The number of tasks is {} after deployment, but only 1 was expected".format(len(tasks))
```

Comment:
Why is this necessary here? We are not testing deployments.

Reply:
it asserts the valid start of the tests... it can be removed if you have strong objections.

Reply:
btw. we don't have to use format anymore since we are on python 3.6 and can use interpolation 🎉